modules/consul/instance.tf

```diff
@@ -29,9 +29,17 @@ resource "null_resource" consul_cluster {
   }
 
   provisioner "file" {
-    source      = "keys"
+    source      = "keys/${count.index}"
     destination = "/home/ubuntu/keys"
   }
+   provisioner "file" {
+    source      = "keys/ca.pem"
```

Comment:
try to use https://www.terraform.io/docs/providers/template/d/file.html and don't store keys locally

Reply:
He tried to do it below. How do you see this approach especially here?